src/main.js

```diff
@@ -74,15 +76,31 @@ const createWindow = async () => {
       ],
     },
   });
-
   const menubar = Menu.buildFromTemplate(
     menuTemplate(mainWindow, ELECTRON_DEV_MODE)
   );
   Menu.setApplicationMenu(menubar);
 
-  // and load the index.html of the app.
+  splashScreen = new BrowserWindow({
```

Comment:
I wonder if moving this to the very start of this function would make it show up any faster?

Reply:
Great idea, I think that helped!